metadata-load/src/test/java/uk/ac/ebi/ampt2d/metadata/importer/MetadataImporterMainApplicationDBTest.java

```diff
@@ -56,13 +57,17 @@
     @Autowired
     private SampleRepository sampleRepository;
 
+    @Autowired
+    private ReferenceSequenceRepository referenceSequenceRepository;
+
     @Test
     @Category(OracleDbCategory.class)
     public void run() throws Exception {
         metadataImporterMainApplication.run(new DefaultApplicationArguments(
                 new String[]{"--accessions.file.path=analysis/EgaAnalysisAccessions.txt"}));
         assertEquals(2, studyRepository.count());
         assertEquals(6, analysisRepository.count());
+        assertEquals(0, referenceSequenceRepository.count()); // No analysis has a proper refSeq accession
```

Comment:
If by `refSeq` you mean `reference sequence` please state it explicitly, because NCBI have a service called RefSeq and this could be very confusing.